### DISCUSSION TITLE
Is AlpineJS still really necessary?

### DISCUSSION THREAD

**Discussion by pyladune** [petalframework/petal_components#207] [Q&A]
With new abilities of LiveView to execute JS command when absolutely necessary, we can wonder if AlpineJS is still required ? Since it comes with consequents pitfalls in terms of security, is still a trivial thing to desactivate it using petal.build ?

Thanks

**Comment by mplatts** [comment]
AlpineJS is optional. It's there for use in dead views for a couple of components.. eg 

Dead views:
```
<.accordion js_lib ="alpine_js">
```

Live views:
```
<.accordion js_lib ="live_view_js">
```
